src/model/virtual_filters/cue_vfilter.py

```diff
@@ -0,0 +1,25 @@
+from model import Filter
```

Comment:
Da fehlt ein commentar und das encoding

Reply:
I don't get it why we're supposed to change the encoding from UTF-8 to UTF-8 on every file but it is now done for this one as well.